docs/guides/Quick Start/01-Introduction.md

```diff
@@ -0,0 +1,134 @@
+# Quick Start Guide
+
+## Overview
+
+Shopware Admin API allows web services to perform administrative tasks. This guide is designed to help you understand the Admin API, its endpoints, and its usage. You will get to learn everything about this API.
+
+* Authentication as API user
+
+* Products 
+
+    * Product creation
+
+    * Categories
+
+    * Dynamic product groups
+
+    * Reviews
+
+    * Cross-selling
+
+    * Price
+
+* Orders
+
+    * Order
+
+    * Line item
+
+    * Order tarnsactions
+
+* Customer
+
+    * Customer creation and groups
+
+* Content
+
+    * CMS PAGES (sec, blocks, slots)
+
+    * CMS Extension plugin (Scroll navigation, Block visibility, Quick view)
```

Comment:
Is this part of the guides?

Reply:
Yes but will be adding this information in the next draft